src/redis_stack/probabilistic_data_structures.md

```diff
@@ -1,27 +1,39 @@
-In very broad terms probabilistic data structures (PDS) allow us to get to a "close enough" result in a much shorter time and by using significantly less memory.
+This tutorial will demonstrate Redis Stack's probabilistic data structure capabilities using the bike shop use case.
 
-Redis Stack supports 4 of the most famous PDS:
-- Bloom filters
-- Cuckoo filters
-- Count-Min Sketch
+Redis Stack supports the following probabilistic data structures:
+
+- Bloom filter
+- Cuckoo filter
+- Count-min sketch
 - Top-K
+- t-digest
+
+Probabilistic data structures, in general, provide results that are "close enough" in a much shorter time and by using significantly less memory than other data types such as sets or hashes. Here, you'll learn how to use a Bloom filter.
```

Comment:
shorter time -> faster?

Reply:
"significantly less memory" - this is not just for providing results, but in general for storing the data (sublinear space)

Reply:
"close enough" is not a good description. e.g., for Bloom filter - the result is a Boolean - not a numeric.